.github/workflows/android.yaml

```diff
@@ -50,3 +50,13 @@ jobs:
             - name: Run Build
               run: |
                   scripts/build/gn_build.sh
+                  rsync -a out/lib/*.jar src/android/CHIPTool/app/libs
```

Comment:
should we split steps into separately named items for easier following? We did already split bootstrap for example.

A large list of commands is harder to follow since output is mixed with the commands themselves.

Reply:
or maybe add a script in scripts directory (which by the way could be used to build the app locally) and run it here?

Reply:
@andy31415 Updated.

@Damian-Nordic Sounds good to me. But we need to update the Android App for new build steps. I would prefer we do it in a separate small PR, this PR is for adding current Android App build steps to GH to make sure it works.

Reply:
@Damian-Nordic Added the script for building Android App, please re-review.